build/config/sanitizers/sanitizers.gni

```diff
@@ -18,3 +18,5 @@ declare_args() {
   # GCS.
   use_prebuilt_instrumented_libraries = false
 }
+
+use_fuzzing_engine = false
```

Comment:
Shouldn't this go in the `declare_args` block?

Reply:
`true` isn't supported without more buildroot work, so it can't be an argument in our build at this point. If someone wants to make it an argument, they'll need to do follow-up work.